packages/grafana-data/src/themes/palette.ts

```diff
@@ -41,4 +41,8 @@ export const palette = {
   greenLightText: '#0A764E',
   orangeLightMain: '#FAD34A',
   orangeLightText: '#8A6C00',
+
+  //fusebit
+  fusebitLightBlue: 'rgba(215, 229, 255, 0.4)',
```

Comment:
@MikeFuster Maybe we can do this change?

```
fusebit: {
   lightBlue,
   white
}
```

@bennbollay What do you think about making this change here?